src/test/resources/META-INF/mods.toml

```diff
@@ -175,8 +175,6 @@ modId="remove_tag_datagen_test"
 [[mods]]
 modId="tag_based_tool_types"
 [[mods]]
-modId="check_spawn_event_test"
-[[mods]]
```

Comment:
Noting for others: the test mod class for this was removed by #9133, which forgot to remove this entry.